src/connecpy/async_client.py

```diff
@@ -97,26 +98,28 @@ async def _make_request(
                     url=self._address + url, content=request_data, **kwargs
                 )
 
-            resp.raise_for_status()
```

Comment:
Because of this, I don't think the `else` on line 106 was ever triggered so we never parsed the JSON response, the only possible error was `Unavailable`